### DISCUSSION TITLE
Which VCard versions are supported?

### DISCUSSION THREAD

**Discussion by sphh** [Kozea/Radicale#1375] [Q&A general] [answered]
I am referring to #832 (DavDroid: Radicale no VCard/4 support). There the problem was solved by implementing a stricter standard on the clients side.

This still leaves me wonder, which VCard versions are supported. And why the Radicale server does not advertise the supported version, as mentioned in 's comment @n00b42 https://github.com/Kozea/Radicale/issues/832#issuecomment-428575642?

1. Please could you specify the supported VCard versions.
2. If the VCard version > 3.0 are supported, please could you let Radicale advertise these versions (it might also be a good idea to advertise the version 3.0 and not relying on the automatic fall back).

**Comment by tasn** [comment]
I did a bit of research, and the reason why Radicale doesn't support vCard 4.0 is because its dependency `vobject` doesn't support 4.0. You can take a look here for more info: https://github.com/eventable/vobject/issues/106

It does however say that reading v4 works. I wonder if that is sufficient for v4 support in Radicale.

**Reply by da4089** [reply]
vObject intends to support vCard 2.1, 3.0, and 4.0 in the near future.  Work on complete 4.0 support will begin soon.

**Reply by antonskalkin73** [reply]
Can you tell us about the current status of work on the implementation of new versions of VCARD?

**Reply by pbiering** [reply, marked as answer]
There is a PR open: https://github.com/py-vobject/vobject/pull/124
And _Radicale_ is prepared since 3.6.0 assuming vobject >= 1.0.0 has it included and also advertise 3.0 for now.
